src/vue/backend/src/views/guided-setup/ListingsIdx.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+    <GuidedSetupContentCard
+        :cardTitle="cardTitle"
+        :steps="guidedSetupSteps"
+        :relatedLinks="links"
+        @back-step="goBackStep"
+        @skip-step="goSkipStep"
+        @continue="goContinue">
+        <template v-slot:controls>
+            <ListingsIdx/>
+        </template>
+    </GuidedSetupContentCard>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex'
+import ListingsIdx from '@/templates/impressListingsIdxContent.vue'
+import GuidedSetupContentCard from '@/templates/GuidedSetupContentCard.vue'
+export default {
+    components: {
+        ListingsIdx,
+        GuidedSetupContentCard
+    },
+    computed: {
+        ...mapState({
+            enabled: state => state.listingsSettings.enabled,
+            guidedSetupSteps: state => state.progressStepper.guidedSetupSteps
+        })
+    },
+    methods: {
+        ...mapActions({
+            progressStepperUpdate: 'progressStepper/progressStepperUpdate',
+            saveListingsIdxSettings: 'listingsSettings/saveListingsIdxSettings'
+        }),
+        goBackStep: function () {
```

Comment:
In an effort to keep our code DRY - why not leverage Vue's [mixins](https://vuejs.org/v2/guide/mixins.html)? For some of these methods you could just pass arguments for the `path` and even the async dispatch call to store.

See our own [tabbed mixin](https://github.com/idxbroker/wordpress-plugin/blob/release/3.0.0/src/vue/backend/src/mixins/Tabbed.js) in this project.

Reply:
Code is very clean otherwise! 👍 😎